docs/howtos/create_custom_appcharts.md

```diff
@@ -84,7 +88,7 @@ Once you modified the chart to your needs, use the following command to package
 changed chart into a tarball:
 
 ```
-$ helm package helm-charts/chart/application
+helm package helm-charts/chart/application
```

Comment:
Not sure if we should change this to `helm package chart/application` since, if I am not wrong, it understands the user is at `/epinio/helm-charts` already

Reply:
Yes, path context is important. Will expand this part.

Reply:
Done with commit 7115b3be85fb0438a6b20f14a45de84b2f93d5dc